lib/beacon_web/live/page_management/page_live/page_editor_live.html.heex

```diff
@@ -31,6 +31,7 @@
           <%= checkbox :opts, :publish, value: false %> Publish on Save
           <div>
             <%= submit "Save", phx_disable_with: "Saving..." %>
+            <%= live_patch "Cancel", to: "/page_management/pages" %>
```

Comment:
Let's keep this relative so we can better support the page management residing in different paths
```suggestion
            <%= live_patch "Cancel", to: "../pages" %>
```

Reply:
Looks like `live_patch` on the `Edit` link in page_manager is not updating the url path to page_editor, so the `../` goes back to `/pages`.  I think we need to use `live_redirect` to get to page editor or use `./`?

Reply:
https://user-images.githubusercontent.com/2897340/190716561-ef723275-f0d7-4d53-b48d-040c40bffc20.mov

Reply:
Wait, isn't `live_patch` supposed to push a new state?
If you notice in the above video, I can't even use the back button